packages/api-client/README.md

```diff
@@ -112,9 +128,9 @@ You can use that to display a better error message.
 
 The client also include a query builder.
 
-Start with one the entry point :
+Start with one the entry point:
 
-- `fromCatalog()` : query the domain catalog
+- `fromCatalog()`: query the domain catalog
```

Comment:
I think the operation is the same for all entry points. So, we must use the same verb. Let's use "access".